### PR TITLE
release notes 5.2

### PR DESCRIPTION
**Pull request by rrelyea**
first draft of release notes.

@karann-msft  - please run with this PR...as I'll be OOF most of the time between now and release.

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[623bd80](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/623bd80de8bf8104762b1b77840bff7db20d23c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907111442502565-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IbWwFME0iEJpUWCyyzaDVQFZpCInGCi76af%2FrrL%2F2xo%3D&st=2019-07-11T14%3A38%3A42Z&se=2019-08-11T14%3A43%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f9aef96](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/f9aef96442525eae55453adda4f28203f0577bca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907111716436941-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=YjMW%2F5dl46c4RIcVoAyRmCwniO7sRKeTV8J2Z%2BL7SPg%3D&st=2019-07-11T17%3A12%3A47Z&se=2019-08-11T17%3A17%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by nkolev92** [comment]
I removed https://github.com/NuGet/Home/issues/8156 from the notes.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6d10283](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/6d10283e1306f1f8290389036e0f89a8f6c38d2e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907112005182961-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ICJ2HyfYqRFjNDVe%2BdzLoFCzuq3I5QRWxvYr0KQ%2Bkbs%3D&st=2019-07-11T20%3A01%3A23Z&se=2019-08-11T20%3A06%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cf96546](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/cf9654614bf7b3df27e731c2d5c66f4374a6e99d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907192207240847-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nexHxMfG0JEF98xskf32180Mf2%2B3NxXISWRtRL8XUr8%3D&st=2019-07-19T22%3A03%3A18Z&se=2019-08-19T22%3A08%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.2-RTM.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ms.topic: conceptual
+---
+
+# NuGet 5.2 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.2.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.1](https://visualstudio.microsoft.com/downloads/) | [2.1.70X](https://dotnet.microsoft.com/download/dotnet-core/2.1)<sup>1</sup>, [2.2.30X](https://dotnet.microsoft.com/download/dotnet-core/2.2)<sup>2</sup> |
```
Comment (karann-msft) [inline docs/release-notes/NuGet-5.2-RTM.md]:
```suggestion
| [**5.2.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.1](https://visualstudio.microsoft.com/downloads/) | [2.1.80X](https://dotnet.microsoft.com/download/dotnet-core/2.1)<sup>1</sup>, [2.2.40X](https://dotnet.microsoft.com/download/dotnet-core/2.2)<sup>2</sup> |
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[dc6dac7](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/dc6dac72c5dbd545199c9a8c8d729009226ef9fd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907192216546566-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hgTHZqPV2CKlPWoWeNRnS1d9tiswIZInjapOjuB9GD8%3D&st=2019-07-19T22%3A12%3A53Z&se=2019-08-19T22%3A17%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[427edea](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/427edea07b69d87e784bdd7bf15b1c05bfcc6f5e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907192219368410-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=S7Fv721%2FqsRVD5D3qLk6ahoMi1VikOvpFDjIxlUV26w%3D&st=2019-07-19T22%3A15%3A38Z&se=2019-08-19T22%3A20%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2b8b011](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1527/commits/2b8b0114ae8cf7d242092de630c820cf30be44ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.2-RTM.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotes-5.2/docs/release-notes/NuGet-5.2-RTM.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.2-rtm?branch=pr-en-us-1527) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907192250288848-1527%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xGi5CGxpBHIH5HTf8zHuN0UjUQUBrFY5Tlh%2Bd4N5pTM%3D&st=2019-07-19T22%3A46%3A23Z&se=2019-08-19T22%3A51%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**